cts/cli/regression.crm_mon.exp

```diff
@@ -5,7 +5,7 @@ Cluster Summary:
   * Last updated:
   * Last change:
   * 5 nodes configured
-  * 27 resource instances configured (4 DISABLED)
+  * 32 resource instances configured (4 DISABLED)
```

Comment:
This is hopefully unrelated and can be addressed separately, but why did the instance count go up by 5? There are only 2 instances of the new clone.

Reply:
Did I screw something up in the definition of promotable-clone in cts/cli/crm_mon.xml?

Reply:
I just noticed "5 nodes configured" above this, that explains the instance count -- now the mystery is why does it think there are 5 nodes